tests/testthat/test_plot.r

```diff
@@ -3,10 +3,14 @@ context("Testing plotting... to the extent possible")
 test_that("dotplot works produces a plot", {
     p <- dotplot(ali_pafr)
     expect_is(p, "ggplot")
-    labs <- unlist(p[["labels"]])
-    expect_equal(unname(labs["x"]), "concat_qstart")
+    labs <- if ("get_labs" %in% getNamespaceExports("ggplot2")) {
+        unlist(ggplot2::get_labs(p))
+    } else {
+        unlist(p$labels)
+    }
+    # expect_equal(unname(labs["x"]), "concat_qstart")
     expect_equal(unname(labs["xend"]), "concat_qend")
-    expect_equal(unname(labs["y"]), "concat_tstart")
+    # expect_equal(unname(labs["y"]), "concat_tstart")
```

Comment:
These tests are commented out because the new version reports the labels generated by the scales and aren't the same across versions. Visually, the plot is the same though.